src/RestServer/Controllers/v1/WalletController.cs

```diff
@@ -0,0 +1,646 @@
+        /// </summary>
+        /// <param name="sessionId" example="066843daf5ce45aba803587780998cdb">Session Id of the open/created wallet.</param>
+        /// <param name="scriptHash" example="0xed7cc6f5f2dd842d384f254bc0c2d58fb69a4761">ScriptHash of the wallet address.</param>
+        /// <returns>A export key object.</returns>
+        /// <response code="200">Successfully exported the key.</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("{session:required}/export/{address:required}", Name = "WalletExportKeysByAddressOrScripthash")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletExportKeyModel))]
+        public IActionResult WalletExportKey(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromRoute(Name = "address")]
+            UInt160 scriptHash)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var account = wallet.GetAccount(scriptHash);
+            if (account == null)
+                throw new WalletException($"Account {scriptHash} doesn\'t exist.");
+            var key = account.GetKey();
+            return Ok(new WalletExportKeyModel
+            {
+                ScriptHash = account.ScriptHash,
+                Address = account.Address,
+                Wif = key.Export(),
+            });
+        }
+
+        /// <summary>
+        /// Create a new address in the wallet with an optional private key.
+        /// </summary>
+        /// <param name="sessionId" example="066843daf5ce45aba803587780998cdb">Session Id of the open/created wallet.</param>
+        /// <param name="model"></param>
+        /// <returns>Wallet address object.</returns>
+        /// <response code="200">Successfully created address.</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpPost("{session:required}/address/create", Name = "WalletCreateAddress")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WalletAddressModel))]
+        public IActionResult WalletCreateNewAddress(
+            [FromRoute(Name = "session")]
+            Guid sessionId,
+            [FromBody]
+            WalletCreateAccountModel model)
+        {
+            if (WalletSessions.ContainsKey(sessionId) == false)
+                throw new KeyNotFoundException(sessionId.ToString("n"));
+            var session = WalletSessions[sessionId];
+            session.ResetExpiration();
+            var wallet = session.Wallet;
+            var account = model.PrivateKey == null || model.PrivateKey.Length == 0 ?
+                wallet.CreateAccount() :
+                wallet.CreateAccount(model.PrivateKey);
```

Comment:
```suggestion
            var account = Wallet.GetPrivateKeyFromWIF(model.Wif) == null || model.Wif.Length == 0 ?
                wallet.CreateAccount() :
                wallet.CreateAccount(Wallet.GetPrivateKeyFromWIF(model.Wif));
```
It's not easy for a user to know his byte[] privatekey, we're all using Wif format privatekey now in Dapp wallets.